settings.gradle

```diff
@@ -25,10 +25,14 @@ gradle.ext.openwhisk = [
 
 gradle.ext.scala = [
     version: '2.12.7',
-    compileFlags: ['-feature', '-unchecked', '-deprecation', '-Xfatal-warnings', '-Ywarn-unused-import']
+    depVersion  : '2.12',
+    compileFlags: ['-feature', '-unchecked', '-deprecation', '-Ywarn-unused-import']
```

Comment:
Here, i removed `''-Xfatal-warnings''`, just want to make `./gradlew --console=plain :tests:test --tests="system.basic.WskCliActivationTests"` test case passed and make this pr merged. 

Anyway, the deprecates issue existed, so i created a issue here: https://github.com/apache/openwhisk-cli/issues/518